documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1737,17 +1672,176 @@ Typically, you configure the {prodname} SQL Server connector in a `.json` file u
 endif::community[]
 
 ifdef::product[]
+To deploy a {prodname} SQL Server connector, you add the connector files to Kafka Connect, create a custom container to run the connector, and then add connector configuration to your container.
+For details about deploying the {prodname} SQL Server connector, see the following topics:
+
+* xref:deploying-debezium-sql-server-connectors[]
+* xref:sql-server-connector-properties[]
+endif::product[]
+
+ifdef::product[]
+// Type: procedure
+// ModuleID: deploying-debezium-sql-server-connectors
+[[sql-server-deploying-a-connector]]
+=== Deploying {prodname} SQL Server connectors
+
+To deploy a {prodname} SQL Server connector, you must build a custom Kafka Connect container image that contains the {prodname} connector archive, and then push this container image to a container registry. 
+You then need to create the following custom resources (CRs):
+
+* A `KafkaConnect` CR that configures your Kafka Connector and that specifies the name of the image that you created to run your {prodname} connector.
```

Comment:
"that configures Kafka Connect" (i.e. not the connector).

Reply:
Oh. If _Connect_ is correct, then that error wasn't picked up in the review of the PG doc last year. If you can confirm, I'll apply the update across all of the connectors.

Reply:
Yes, this first CR is about Kafka Connect. There's two things involved in general:

* _Kafka Connect_, which is the runtime environment upon which the actual connector is deployed
* The actual Debezium _connectors_, which are deployed to KC and perform the actual CDC work

I just checked, indeed that's not right in the PG docs, too. Look for "Kafka Connector" (with upper-case "C"), that never makes sense. Either "Kafka Connect", or "connectors" (that said, there is `KafkaConnector` as the CRD kind processed  by the connector operator).